src/main/groovy/org/mockito/release/notes/model/Commit.java

```diff
@@ -1,12 +1,14 @@
 package org.mockito.release.notes.model;
 
+import org.json.simple.Jsonable;
+
 import java.io.Serializable;
 import java.util.Collection;
 
 /**
  * VCS change, a commit
  */
-public interface Commit extends Serializable{
+public interface Commit extends Jsonable, Serializable {
```

Comment:
Yes, I tried to remove extending Jsonable from interface, but our implementation classes uses interfaces instead of other classes implementation.